Makefile

```diff
@@ -111,8 +111,11 @@ venv-upgrade-dev:  ## Upgrade all default+dev dependencies defined in setup.cfg
 ##########################################
 # Targets specific to Docker installations
 
+env.d/development:
+	cp env.d/development.dist env.d/development
```

Comment:
If I'm not mistaken this would squash an existing file which is not what we want. You should only copy it if the development file does not exist yet?

Reply:
Yes you are right but this task should not be run alone. It's only ouroise for now is to be run only id the file `env.d/development` doest not exists when you run the `bootstrap` task. This is why I didn't comment it to not appear when you run `make help` task.

But maybe checking if the file doest not already exists can be a good idea

Reply:
In fact no, if the file already exists make will respond `make: 'env.d/development' is up to date.` so nothing more is needed here :-)

Reply:
Ok! I take it that you checked :stuck_out_tongue:

Reply:
Actually, are you sure that it still respects you env.d/development file if you modify it? Because make is supposed to re-execute the target when the file changes? so in your case it would run the command line again and erase it :scream: ?

Reply:
once created it will not be erase nor its content modified

Reply:
By default a `Makefile` target is meant to point to a file and the purpose of this target is to create this file. It means that the target won't be re-executed if the file already exists. Remember the old days were Makefiles were used to compile C code:

```Makefile
foo.o : foo.c defs.h       # module for twiddling the frobs
        cc -c -g foo.c
```

Reference: https://www.gnu.org/software/make/manual/make.html#Rules